src/main/resources/jseont_sitemap/xml/sitemap.index.jsp

```diff
@@ -42,24 +43,21 @@
                 <c:set var="currentLanguage" value="${renderContext.site.language}"/>
                 <c:forEach var="lang" items="${languages}">
                         <c:if test="${not (currentLanguage eq lang) and not functions:contains(inactiveLanguages, lang)}">
-                                <c:set var="replaced" value="/sites/"/>
-                                <c:set var="replacee" value="/${lang}/sites/"/>
-                                <c:url value="${siteMapPath}.xml" var="languageResource"/>
+                                <c:url value="${url.getBase(lang.toString())}${siteMapPath}.xml" var="languageResource"/>
                                 <sitemap>
-                                        <loc>${url.server}${fn:replace(languageResource, replaced, replacee)}</loc>
+                                        <loc>${url.server}${languageResource}</loc>
                                 </sitemap>
                         </c:if>
                 </c:forEach>
 
                 <%--  Separate sitemaps for jseont:sitemapResource node option --%>
-                <%-- TODO: replace jnt:page to jseont:sitemapResource once nodetype is available --%>
                 <jcr:jqom var="additionalMaps">
-                        <query:selector nodeTypeName="jnt:page" selectorName="stmp"/>
+                        <query:selector nodeTypeName="jseont:sitemapResource" selectorName="stmp"/>
```

Comment:
missing a check to add nodes only when parent `jseomix:sitemapResource` has `createSitemap` attribute enabled

Reply:
Yes, you're right!